clik/external/hal_cpu/include/cpu_hal.h

```diff
@@ -25,6 +25,9 @@
 
 #include "hal.h"
 
+#define HAL_CPU_WI_MODE 0
+#define HAL_CPU_WG_MODE 1
```

Comment:
Due to the way the preprocessor works, if `HAL_CPU_MODE` is not defined, `#if HAL_CPU_MODE == HAL_CPU_WI_MODE` will be true and the `#error` that you added will never fire. Can you change the values from 0/1 to 1/2? (Pre-approving)